### PR TITLE
added some love to the about section :)

### PR DESCRIPTION
**Pull request by ncoelho**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/khfpt2wgkr304m94wpuq8pmnuw

[download apk](http://filestorage.bluewallet.io:1488/1585679424.apk)

**Comment by Overtorment** [comment]
We need to add License section (MIT), and probably full text of MIT license

**Comment by marcosrdz** [comment]
LGTM

File: ios/Podfile.lock [hunk 1]
```diff
@@ -539,4 +539,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 932f5bffd3ddea34b78c9ed0921a6f8cccf442b5
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.7.5
```
Comment (Overtorment) [inline ios/Podfile.lock]: @marcosrdz this is not okay, right?

File: ios/Podfile.lock [hunk 2]
```diff
@@ -520,7 +520,7 @@ SPEC CHECKSUMS:
   RNCAsyncStorage: 8539fc80a0075fcc9c8e2dff84cd22dc5bf1dacf
   RNDefaultPreference: 12d246dd2222e66dadcd76cc1250560663befc3a
   RNDeviceInfo: 12faae605ba42a1a5041c3c41a77834bc23f049d
-  RNFS: 90d1a32d3bc8f75cc7fc3dd2f67506049664346b
+  RNFS: 2bd9eb49dc82fa9676382f0585b992c424cd59df
```
Comment (Overtorment) [inline ios/Podfile.lock]: @marcosrdz do you approve this?

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xb2g7bhk7cx7cjrwag4g49ef1w

[download apk](http://filestorage.bluewallet.io:1488/1585680816.apk)

File: ios/Podfile.lock (hunk 2 above)
Comment (marcosrdz) [reply ios/Podfile.lock:523]: Approved.

**Pull request merged**